magpie/ui/management/templates/tree_scripts.mako

```diff
@@ -42,16 +42,15 @@ li.Collapsed {
     <ul>
      %for key in tree:
         <div class="clear"/>
-        <form id="resource_${tree[key]['id']}" action="${request.path}" method="post">
+        <form id="resource_${tree[key]['id']}_${tree[key].get('remote_id', '')}" action="${request.path}" method="post">
```

Comment:
Has this been validated to work with other kind of services that do not employ the cron?
Does `render_tree` still work with other pages that might not have `_remoteID` in their id?